module/actor/sheets/WitcherCharacterSheet.js

```diff
@@ -305,4 +307,11 @@ export default class WitcherCharacterSheet extends WitcherActorSheet {
             }
         }).render(true)
     }
+
+    async _repairItem(event) {
+        let itemId = event.currentTarget.closest(".item").dataset.itemId
+        let item = this.actor.items.get(itemId)
+
+        await RepairSystem.process(this.actor, item)
```

Comment:
I think repairing an item should live on the item.

so calls should look like ``item.repair()`` etc